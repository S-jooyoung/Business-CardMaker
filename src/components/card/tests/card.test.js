// import dependencies
import React from "react";

// import custom-component
import Card from "../card";

// import snapshot-testing methods
import renderer from "react-test-renderer";

describe("Card", () => {
  let cardComponent;
  let card = {
    name: "test",
    company: "test",
    title: "test",
    email: "jooyoung.dev@gmail.com",
    message: "test",
    theme: "dark",
    fileURL: "",
  };

  beforeEach(() => {
    cardComponent = <Card card={card} />;
  });

  it("render", () => {
    const component = renderer.create(cardComponent).toJSON();

    expect(component).toMatchSnapshot();
  });
});
