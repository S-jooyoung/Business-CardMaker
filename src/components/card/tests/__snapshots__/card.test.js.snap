// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render 1`] = `
<li
  className="card dark"
>
  <img
    alt="profile photo"
    className="avatar"
    src="/images/default_logo.png"
  />
  <div
    className="info"
  >
    <h1
      className="name"
    >
      test
    </h1>
    <p
      className="company"
    >
      test
    </p>
    <p
      className="title"
    >
      test
    </p>
    <p
      className="email"
    >
      jooyoung.dev@gmail.com
    </p>
    <p
      className="message"
    >
      test
    </p>
  </div>
</li>
`;
